src/GeneratedMapper/Builders/Base/BuilderBase.cs

```diff
@@ -64,33 +65,15 @@ protected void WriteOptionalNullableEnablePragma(IndentedTextWriter indentWriter
                 indentWriter.WriteLine();
             }
         }
-        protected void WriteOpenNamespaceAndStaticClass(IndentedTextWriter indentWriter, string extraNamespaceName, string className)
+        protected IDisposable WriteOpenNamespaceAndStaticClass(IndentedTextWriter indentWriter, string extraNamespaceName)
```

Comment:
I think `WriteOpenNamespaceAndStaticClass` should be renamed to `WriteOpenNamespace` as it does not output the static class anymore.